packages/integration-sdk-cli/src/commands/collect.ts

```diff
@@ -60,10 +82,43 @@ export function collect() {
         },
         {
           enableSchemaValidation,
+          useDependenciesCache: options.useDependenciesCache,
           graphObjectStore,
         },
       );
 
       log.displayExecutionResults(results);
     });
 }
+
+/**
+ * Sets environment variable JUPITERONE_CACHE_DIRECTORY
+ * to be used in reading & writing to cache.
+ * @param useDependenciesCache
+ */
+function setupCacheDirectory(useDependenciesCache) {
+  process.env.JUPITERONE_CACHE_DIRECTORY = path.resolve(
+    useDependenciesCache,
+    '.j1-cache',
+  );
+
+  log.info(
+    `Set dependencies cache location to ${process.env.JUPITERONE_CACHE_DIRECTORY}`,
+  );
+}
+
+/**
+ * When no filepath is specified, the .j1-integration directory
+ * is copied to .j1-cache
+ */
+async function copyToCache() {
+  const graphDirectory = path.join(getRootStorageDirectory(), 'graph');
+  if (fs.ensureDir(graphDirectory)) {
```

Comment:
This seems like it is an async function? See: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/fs-extra/index.d.ts#L45

Can we install `@types/fs-extra` as a dev dep?